app/forms/import_form.rb

```diff
@@ -0,0 +1,24 @@
+class ImportForm
+  extend ActiveModel::Naming
+  include ActiveModel::Conversion
+  include ActiveModel::Validations
+
+  ATTRIBUTES = [:url, :preview]
+  attr_accessor *ATTRIBUTES
+
+  validates_presence_of :url
+
+  def persisted?
+    false
+  end
+
+  def preview?
+    preview == "1"
+  end
+
+  def initialize(attributes = {})
```

Comment:
If you include ActiveModel::Model instead of the includes above, I think you get this for free.